packages/angular/src/components/axis/axis.component.ts

```diff
@@ -191,8 +160,8 @@ export class VisAxisComponent<Datum> implements AxisConfigInterface<Datum>, Afte
   }
 
   private getConfig (): AxisConfigInterface<Datum> {
-    const { duration, events, attributes, x, y, id, color, xScale, yScale, excludeFromDomainCalculation, position, type, fullSize, label, labelFontSize, labelMargin, labelColor, gridLine, tickLine, domainLine, minMaxTicksOnly, tickFormat, tickValues, numTicks, tickTextFitMode, tickTextWidth, tickTextSeparator, tickTextForceWordBreak, tickTextTrimType, tickTextFontSize, tickTextAlign, tickTextColor, tickPadding } = this
```

Comment:
@rokotyan Why are the properties inherited from `XYComponentConfigInterface` being excluded if we are keeping the ones from `ComponentConfigInterface`?

Reply:
That only affects  some of the components (Axis, Tooltip, Crosshair, Brush, FreeBrush). Those properties are technically useless for auxiliary components.